lib/node/runner/broadcaster_test.go

```diff
@@ -13,7 +13,7 @@ func TestConnectionManagerBroadcaster(t *testing.T) {
 	recv := make(chan struct{})
 	nr, _, cm := createNodeRunnerForTesting(3, conf, recv)
 
-	nr.StartStateManager()
+	nr.startStateManager()
```

Comment:
This commit title does not match its content. Maybe you meant "Make `startStateManager` private" ?